kagenti/installer/app/utils.py

```diff
@@ -26,6 +26,42 @@
 
 console = Console()
 
+def get_latest_tagged_version(github_repo, fallback_version) -> str:
+    """Fetches the latest version tag of the component from GitHub releases.
+
+    Args:
+        github_repo (str): The GitHub repository path of the component.
+        fallback_version (str): The fallback version to return if fetching fails.
+
+    Returns:
+        str: The latest version tag or the fallback version.
+    """
+    try:
+        result = subprocess.run(
+            [
+                "git", "ls-remote", "--tags", "--sort=-version:refname",
+                github_repo,
+            ],
+            capture_output=True,
+            text=True,
+            check=True,
+            timeout=30
+        )
+
+        lines = result.stdout.strip().split('\n')
+        for line in lines:
+            if line and 'refs/tags/' in line:
+                # Extract tag name
+                tag = line.split('refs/tags/')[-1]
+                if '^{}' not in tag:  # Exclude annotated tags
+                    return tag
+
+        print("Could not find tag_name in the response. Using fallback version.")
+        return fallback_version
+    except subprocess.CalledProcessError as e:
+        print(f"Error fetching latest version: {e}. Using fallback version.")
```

Comment:
I suggest using console.log() instead of print - e.g. `console.log(f"[bold red]Error fetching latest version: {e}. Using fallback version.[/bold red]")`